src/ensembl/src/content/app/help/components/help-menu/HelpMenu.tsx

```diff
@@ -40,6 +42,8 @@ const HelpMenu = (props: Props) => {
   const [submenuItems, setSubmenuItems] = useState<MenuItem[] | null>(null);
   const clickedMenuRef = useRef<number | null>(null);
 
+  const { trackMegaNavOpen } = useHelpAppAnalytics();
```

Comment:
If we are also tracking clicks on the links that do not open the mega nav (such as "Overview") with this function, then let's at least not confuse ourselves with its name – it is not tracking the opening of the mega nav, but rather the clicking on the top-level menu item.